lib/scenic/adapters/postgres.rb

```diff
@@ -195,8 +196,10 @@ def drop_materialized_view(name)
       #   Scenic.database.refresh_materialized_view(:posts, concurrent: true)
       #
       # @return [void]
-      def refresh_materialized_view(name, concurrently: false)
+      def refresh_materialized_view(name, concurrently: false,
+                                    with_dependencies: false)
```

Comment:
I prefer to have signatures on one line, even if they go longer than whatever length, for method definitions. Can you join these?